docs/src/testing/load_testing/load_testing.md

```diff
@@ -76,7 +71,97 @@ echo off | sudo tee /sys/devices/system/cpu/smt/control
 One way of checking this, besides the command above,
 is to open htop, you should see the virtual cores as 'offline'.
 
-### Load Test Client setup
+2. Run this command to increase the file descriptor amount.
+```bash
+ulimit -n 65000
+```
+
+3. Export the variables defined at `~/.env`
+    ```bash
+    export $(cat ~/.env | xargs)
+    ```
+4. Set the env variables for newrelic:
+
+    ```bash
+    export NEW_RELIC_APP_NAME=
+    export NEW_RELIC_LICENSE_KEY=
+    ```
+
+    You will have to ask for them
+
+5. Now you can start the game server with: 
+```sh
+cd game_backend && MIX_ENV=prod iex -S mix phx.server
+```
+   From now on, you can just use: 
+```sh
+MIX_ENV=prod iex -S mix phx.server
+```
+
+### Starting the load test runner
+
+1. Set this env variable: `export SERVER_HOST=game_server_ip:game_server_port`.
+   Where `game_server_ip` is the ip of the load test server, and `game_server_port`,
+   the corresponding port.
+2. Run this command to increase the file descriptor amount.
+```bash
+ulimit -n 65000
+```
+3. Run:
+   ```sh
+       cd ./curse_of_myrra/server/load_test/ && iex -S mix 
+   ``` 
+   this drops you into an Elixir shell from which you'll run the load tests.
+4. From the elixir shell, start the load test with:
+   ```elixir
+   LoadTest.PlayerSupervisor.spawn_players(number_of_players, play_time_in_seconds)
+   ``` 
+
+(You don't have to literally put `user` there. Put the appropiate user based on the server you'll be using).
+
+
+## Load Test Server First Time Setup
+
+This is supposed to be done only once per server. If it was already done, you can skip this step.
+
+1. Check you can log into it with ssh: 
+   ```sh
+   ssh myuser@myrra_load_test_server
+   ```
+2. Then run `exit` and copy the script on this repo under
+   `load_test/setup_game_server.sh` it installs all dependencies, clones the game server and compiles it:
+   ```sh
+   scp load_test/setup_game_server.sh myrra_load_test_server:setup_game_server.sh && ssh myuser@myrra_load_test_server
```

Comment:
Don't we also need to clarify that this file is located in the game_backend repo?

```suggestion
   scp /path_go_game_backend/game_backend/load_test/setup_game_server.sh myrra_load_test_server:setup_game_server.sh && ssh myuser@myrra_load_test_server
```